fzf-tab.zsh

```diff
@@ -306,7 +306,50 @@ _fzf_tab_complete() {
 
 zle -C _fzf_tab_complete complete-word _fzf_tab_complete
 
+__fzfcmd_complete() {
+  [ -n "$TMUX_PANE" ] && [ "${FZF_TMUX:-0}" != 0 ] && [ ${LINES:-40} -gt 15 ] &&
+    echo "fzf-tmux -d${FZF_TMUX_HEIGHT:-40%}" || echo "fzf"
+}
+
+_fzf_feed_fifo() (
```

Comment:
If you copy functions over, give them a different name. Otherwise, when these functions get modified in fzf, the behavior of fzf-tab, and (worse!) of plain fzf will depend on the order in which files are sourced. It would be quite bad if fzf-tab affects the behavior of plan fzf.